syntax = "proto3";

option java_package = "com.interfaces.proto";
option java_outer_classname = "VehicleTypeProto";

enum VehicleType
{
    THREE_WHEELED = 0;
}